### DISCUSSION TITLE
Achievement: How to see how many PR I have done?

### DISCUSSION THREAD

**Discussion by 53845714nF** [community/community#123499] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

If I look at my PRs, there are 28 Merged PRs. 
But I didn't get the Bronze Pull Shark. Do they have to fulfil anything else?

**Comment by IgorD-lab** [comment]
The Pull Shark badge only considers merged pull requests in public repositories.
There's a chance your 28 merged PRs might be in private repositories, which wouldn't count towards the badge.

**Comment by karthickkumarsm** [comment, marked as answer]
To get the Bronze Pull Shark badge on GitHub, you need:

1. At least 2 merged pull requests.
2. Public Repositories: The pull requests must be in public repositories, not your own personal ones.
3. Merged by Others: Someone else should merge your pull requests, not you.

If you have 28 merged pull requests but don't have the badge yet:

- Wait a bit: Sometimes it takes time for the badge to show up.
- Check Repositories: Make sure the pull requests are in public repositories and were merged by someone else.
- Contact Support: If everything seems right and you still don't have the badge, contact GitHub support for help.
